specification/cognitiveservices/data-plane/QnAMaker/preview/v5.0-preview.1/QnAMaker.json

```diff
@@ -0,0 +1,1908 @@
+          "minLength": 1
+        },
+        "language": {
+          "type": "string",
+          "description": "Language of the knowledgebase. Please find the list of supported languages <a href=\"https://aka.ms/qnamaker-languages#languages-supported\" target=\"_blank\">here</a>.",
+          "maxLength": 100,
+          "minLength": 1
+        }
+      }
+    },
+    "FileDTO": {
+      "type": "object",
+      "description": "DTO to hold details of uploaded files.",
+      "additionalProperties": false,
+      "required": [
+        "fileName",
+        "fileUri"
+      ],
+      "properties": {
+        "fileName": {
+          "type": "string",
+          "description": "File name. Supported file types are \".tsv\", \".pdf\", \".txt\", \".docx\", \".xlsx\".",
+          "maxLength": 200,
+          "minLength": 1
+        },
+        "fileUri": {
+          "type": "string",
+          "description": "Public URI of the file."
+        }
+      }
+    },
+    "ReplaceKbDTO": {
+      "type": "object",
+      "description": "Post body schema for Replace KB operation.",
+      "additionalProperties": false,
+      "required": [
+        "qnAList"
+      ],
+      "properties": {
+        "qnAList": {
+          "type": "array",
+          "description": "List of Q-A (QnADTO) to be added to the knowledgebase. Q-A Ids are assigned by the service and should be omitted.",
+          "items": {
+            "$ref": "#/definitions/QnADTO"
+          }
+        }
+      }
+    },
+    "QnADTO": {
+      "type": "object",
+      "description": "Q-A object.",
+      "additionalProperties": false,
+      "required": [
+        "answer",
+        "questions"
+      ],
+      "properties": {
+        "id": {
+          "type": "integer",
+          "description": "Unique id for the Q-A.",
+          "format": "int32"
+        },
+        "answer": {
+          "type": "string",
+          "description": "Answer text",
+          "maxLength": 25000,
+          "minLength": 1
+        },
+        "source": {
+          "type": "string",
+          "description": "Source from which Q-A was indexed. eg. https://docs.microsoft.com/en-us/azure/cognitive-services/QnAMaker/FAQs",
+          "maxLength": 300
+        },
+        "questions": {
+          "type": "array",
+          "description": "List of questions associated with the answer.",
+          "maxLength": 100,
+          "minLength": 1,
+          "items": {
+            "type": "string"
+          }
+        },
+        "metadata": {
+          "type": "array",
+          "description": "List of metadata associated with the answer.",
+          "maxLength": 10,
+          "items": {
+            "$ref": "#/definitions/MetadataDTO"
+          }
+        },
+        "context": {
+          "description": "Context of a QnA",
+          "allOf": [
+            {
+              "$ref": "#/definitions/ContextDTO"
+            }
+          ]
+        },
+        "changedTimeStamp": {
```

Comment:
code has field: LastUpdatedTimestampUTC , 
this is vijaya's new one right?, any change in name field?